components/Header.tsx

```diff
@@ -1,8 +1,46 @@
+import Image from "next/image"
+import { WagmiConfig, createClient } from 'wagmi'
+
+const client = createClient()
+
+function Wallet() {
+  return (
+    <WagmiConfig client={client}>
+      <Profile />
+    </WagmiConfig>
+  )
+}
+
+import { useAccount, useConnect, useDisconnect } from 'wagmi'
+import { InjectedConnector } from 'wagmi/connectors/injected'
+
+function Profile() {
+  const { data } = useAccount()
+  const { connect } = useConnect({
+    connector: new InjectedConnector(),
+  })
+  const { disconnect } = useDisconnect()
+
+  if (data)
+    return (
+      <div>
+        Connected to {data.address}
```

Comment:
Can you shorten the address so that it does not take up so much space in the UI?